### PR TITLE
systemmanager, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: haiku-apps/systemmanager/systemmanager-0.0.1~git.recipe [hunk 1]
```diff
@@ -0,0 +1,52 @@
+SUMMARY="System manager for Haiku"
```
Comment (humdingerb) [inline haiku-apps/systemmanager/systemmanager-0.0.1~git.recipe:1]: All the software in haikuports is "for Haiku" :)

```suggestion
SUMMARY="A tool to manage teams and threads"
```

File: haiku-apps/systemmanager/systemmanager-0.0.1~git.recipe [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+DESCRIPTION="List running teams and explore various team objects (images, threads, etc.). Also \
```
Comment (humdingerb) [inline haiku-apps/systemmanager/systemmanager-0.0.1~git.recipe:2]:
```suggestion
DESCRIPTION="SystemManager shows all running teams and lets you explore various team objects \
(images, threads, semaphores, etc.). It also generates stack traces of selected threads.
```

**Comment by Begasus** [comment]
Build failure on 32bit now (with the extra binaries)

File: haiku-apps/haikuutils/haikuutils-r130~git.recipe [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	makefile_engine
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	"
+
+Utils="PictureView Services SystemManager TraverseApps Users"
+if [ "$effectiveTargetArchitecture" = x86_64 ]; then
+	Utils+=" MsgDump"
+fi
+DeskbarUtils="Services SystemManager Users"
```
Comment (Begasus) [inline haiku-apps/haikuutils/haikuutils-r130~git.recipe:51]: There are meant to be in the menu, the other binaries are meant to be run in Terminal.

**Pull request merged**